cli/core/upgrade.go

```diff
@@ -91,6 +92,7 @@ func runUpgradeCommand(cmd *cobra.Command, args []string) {
 			Instance:        inst,
 			PlatformPackage: platformRef.PackageName,
 			Architecture:    platformRef.Architecture,
+			SkipPostInstall: detectSkipPostInstallValue(),
```

Comment:
Sure! I'll check it out.

Reply:
I looked at it and think it might be necessary to port them on the `upgrade` command, it don't think it should be necessary for `update` since it just downloads the index files.